addons/website/static/src/builder/plugins/options/process_steps_option_plugin.js

```diff
@@ -5,16 +5,18 @@ import { Plugin } from "@html_editor/plugin";
 import { registry } from "@web/core/registry";
 import { connectorOptionParams, ProcessStepsOption } from "./process_steps_option";
 import { WebsiteBackgroundOption } from "./background_option";
+import { withSequence } from "@html_editor/utils/resource";
+import { PROCESS_STEPS } from "@website/builder/option_sequence";
 
 class ProcessStepsOptionPlugin extends Plugin {
     static id = "processStepsOption";
     selector = ".s_process_steps";
     resources = {
         builder_options: [
-            {
+            withSequence(PROCESS_STEPS, {
```

Comment:
```suggestion
            withSequence(after(CONTAINER_WIDTH), {
```
May be you should do something like this.

Reply:
Putting them in `option_squence.js` will make sequencing easy. Example: if i have to place something between the both, it will make things hard and in the end we have to use `option_sequence.js`.